libsql-replication/proto/metadata.proto

```diff
@@ -12,11 +19,12 @@ message DatabaseConfig {
     // maximum db size (in pages)
     uint64 max_db_pages = 4;
     optional string heartbeat_url = 5;
-    optional string bottomless_db_id = 6; 
+    optional string bottomless_db_id = 6;
     optional string jwt_key = 7;
     optional uint64 txn_timeout_s = 8;
     bool allow_attach = 9;
     optional uint64 max_row_size = 10;
     optional bool shared_schema = 11;
     optional string shared_schema_name = 12;
+    DurabilityMode durability_mode = 13;
```

Comment:
Should this be optional?

Reply:
I am wondering if that is needed. Because in the `DatabaseConfig` it is not optional, set to default `Relaxed`. So if we make this optional, for existing DBs, it will be `None` only at the first read. When the `DatabaseConfig` is saved, it will be set to the default value.

For newer DBs, this will always have a default value from `DatabaseConfig` and will never be `None`

WDYT?

Reply:
we should adhere to that: https://protobuf.dev/getting-started/pythontutorial/#extending-a-protobuf

Reply:
that is fair, I will update the PR